detection-rules/attachment_archive_with_exe.yml

```diff
@@ -14,7 +14,7 @@ type: "rule"
 severity: "high"
 source: |
   type.inbound
-  and any(attachments, .file_extension in~ $file_extensions_common_archives
```

Comment:
porque no los dos? also remember `file_type` values might not intersect with that list how you think, so this might introduce some FNs